src/main/java/cursor/hw12/service/LFUCacheService.java

```diff
@@ -0,0 +1,61 @@
+package cursor.hw12.service;
+
+import cursor.hw12.dao.AverageSalaryDao;
+import cursor.hw12.model.Node;
+import java.util.Map;
+
+public class LFUCacheService {
+    private static final int capacity = 10;
```

Comment:
if it's a constant it should be private static final int CAPACITY = 10;

Reply:
Resolved